### PR TITLE
test(ed-cdk): test new distro and static constructs

### PR DESCRIPTION
**Pull request by RyanMulready**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [x] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [x] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

- Move towards using `es-cdk` and new `nuxtStatic` construct.
- Updated confluence [deploy](https://energysage.atlassian.net/wiki/spaces/DSE/pages/208568321/Deploy+Design+System+Documentation) doc with new steps needed
- Disabled `telemetry` option in `nuxt.config`
- Removed `.env` files for now; only `.env.dev` was in use; added `ACCOUNT_ID` in code instead
- Removed `/version/` path from deploy and config; **Thoughts here?**

### 🥼 Testing

- new domain is deployed at https://test.design.energysage.dev/

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.
- [x] I have documented testing approach
- [x] Bump `es-cdk` to stable version when published

### CODE REVIEW

**Comment by tomleo** [comment]
> Removed /version/ path from deploy and config; Thoughts here?

It seems like `<feature-branch>.<sub-domain>.energysage.dev` is a more common scheme in other repos, so I'm in favor dropping the version pathing.

I can imagine a future state where we might want to also have `<semver>.test.design.energysage.dev` for upcoming releases that are pending design-review.

File: es-design-system/aws/package.json [hunk 1]
```diff
@@ -8,11 +8,10 @@
     "build": "echo \"The build step is not required when using JavaScript!\" && exit 0",
     "cdk": "cdk"
   },
-  "devDependencies": {
-    "aws-cdk": "2.22.0"
-  },
   "dependencies": {
-    "aws-cdk-lib": "2.22.0",
-    "constructs": "^10.0.0"
+    "aws-cdk": "^2.66.1",
```
Comment (tomleo) [inline es-design-system/aws/package.json:12]: Can `aws-cdk` be dropped as well?

File: es-design-system/aws/package.json [hunk 2]
```diff
@@ -8,11 +8,10 @@
...
+    "aws-cdk-lib": "^2.64.0",
+    "constructs": "^10.0.0",
+    "es-cdk": "^1.19.0"
```
Comment (tomleo) [inline es-design-system/aws/package.json:15]: 🎉

File: es-design-system/nuxt.config.js [hunk 3]
```diff
@@ -33,14 +33,12 @@ export default {
...
     },
-    router: {
-        base: `/${version}/`,
-    },
     // Development has SSR turned on to test es-vue-base compatibility
     // Production build will generate client only SPA
     ssr: process.env.NODE_ENV === 'development',
     target: 'static',
     modern: true,
+    telemetry: false,
```
Comment (tomleo) [inline es-design-system/nuxt.config.js:41]: 👍

File: es-design-system/package.json [hunk 4]
```diff
@@ -6,15 +6,11 @@
         "dev": "env-cmd -f .env.local nuxt",
         "build": "nuxt generate",
         "build:dev": "npm run generate:dev && npm run deploy:dev",
-        "build:prod": "npm run generate:prod && npm run deploy:prod",
-        "generate:dev": "env-cmd -f .env.dev npm run build",
-        "generate:prod": "env-cmd -f .env.prod npm run build",
-        "bootstrap": "env-cmd -f .env.prod npm --prefix aws run cdk bootstrap",
-        "synth:dev": "env-cmd -f .env.dev npm run synth",
-        "synth:prod": "env-cmd -f .env.prod npm run synth",
+        "generate:dev": "npm run build",
```
Comment (tomleo) [inline es-design-system/package.json:9]: We don't have to address this in this PR, but the npm script names are a little confusing. I would expect "generate" to run `nuxt generate` and "build" to run `nuxt build`, ect.

**Review by tomleo — APPROVED** [review]
👍

File: es-design-system/package.json (hunk 4 above)
Comment (RyanMulready) [reply es-design-system/package.json]: [ab0e276](https://github.com/EnergySage/es-ds/pull/666/commits/ab0e276ec07e3cf366eed33f56b61b58f2e44ec8)

**Pull request merged**